server/src/main/java/org/apache/iotdb/db/mpp/plan/scheduler/FragmentInstanceDispatcherImpl.java

```diff
@@ -268,11 +270,15 @@ private void dispatchRemote(FragmentInstance instance, TEndPoint endPoint)
           }
           break;
         case WRITE:
-          TSendPlanNodeReq sendPlanNodeReq =
-              new TSendPlanNodeReq(
-                  new TPlanNode(instance.getFragment().getPlanNodeTree().serializeToByteBuffer()),
-                  instance.getRegionReplicaSet().getRegionId());
-          TSendPlanNodeResp sendPlanNodeResp = client.sendPlanNode(sendPlanNodeReq);
+          TSendBatchPlanNodeReq sendPlanNodeReq =
```

Comment:
It seems that this change is unnecessary because only one Req is processed in this batch. Maybe we can still use original way to do that

Reply:
I think batch means that there is more than or equal to one. Writing two interfaces in thrift for the same logic results in more redundant code on the sender and receiver...

Reply:
make sense